packages/cli/src/index.ts

```diff
@@ -267,6 +267,7 @@ const main = async () => {
     config,
     authConfig,
     localConfig,
+    localConfigPath,
```

Comment:
Seems like we could just use `localConfig`, instead of passing in a new prop for the path?